docker-compose-full.yml

```diff
@@ -100,6 +107,57 @@ services:
     volumes:
       - ../generic-tooling-source:/opt/generic-tooling-target
 
+  language-server-proxy:
+    image: exercism/language-server-proxy
+    build:
+      context: ../language-servers
+      dockerfile: proxy.Dockerfile
+    ports:
+      - 3023:3023
+    environment:
+      RUBY_LANGUAGE_SERVER_HOST: ruby-language-server
+      RUBY_LANGUAGE_SERVER_PORT: 7658
+      JAVASCRIPT_LANGUAGE_SERVER_HOST: javascript-language-server
+      JAVASCRIPT_LANGUAGE_SERVER_PORT: 2538
+      CSHARP_LANGUAGE_SERVER_HOST: csharp-language-server
+      CSHARP_LANGUAGE_SERVER_PORT: 8291
+    volumes:
+      - ../language-servers:/opt/exercism/language-server-proxy/current
```

Comment:
The path to the proxy's working dir (`/opt/exercism/language-server-proxy/current`) is slightly inconsistent with the other paths used in our Docker images. The tooling invoker, orchestrator and website all use `/usr/src/app`, so it might be nice standardizing on this path for the language servers too. Note that this should be done both here and in the Dockerfiles in the language-servers repo.